zappa/cli.py

```diff
@@ -1464,11 +1465,8 @@ def tabular_print(title, value):
             # There literally isn't a better way to do this.
             # AWS provides no way to tie a APIGW domain name to its Lambda function.
             domain_url = self.stage_config.get('domain', None)
-            base_path = self.stage_config.get('base_path', None)
```

Comment:
Any idea why the base_path is being removed from here?

Reply:
My guess is a bad merge, seeing as I had to remove `base_path` before to merge master into my branch. It looks like you might be taking this PR over -- in which case you're welcome to make that change yourself, or I can.